### DISCUSSION TITLE
Translations priority

### DISCUSSION THREAD

**Discussion by Christophvh** [strapi/strapi#7433] [Legacy Topics]
Hi, thanks for the nice product and building it as open-source.

We are considering using Strapi for our next big project, but the lack of translation support makes it impossible to recommend for our customers. Here in Belgium, almost all medium-sized projects have at least 3 languages. The roadmap suggests that the design phase will be in Q4 of this year. What exactly does the design phase mean and where can we track progress or contribute? I also wonder why there are 2 items on the roadmap planned for Q3 with way fewer votes.

https://portal.productboard.com/strapi/1-public-roadmap/tabs/6-in-progress-design

Draft state (150 upvotes), Api tokens (120 upvotes). But translation support has 860 upvotes. So translation support has almost 8 times more priority for potential users so I was wondering why those other two tasks will be handled first?

**Comment by derrickmehaffy** [comment]
Hi @Christophvh one of the key requirements for the internationalization plugin is the new plugin API (also on the PB: https://portal.productboard.com/strapi/1-public-roadmap/c/72-plugin-api-v4 ) as this is a hard requirement and will take quite a bit of time to develop in parallel to the internationalization, this is the main reason it's scheduled for the end of Q4. A rough ETA I just got internally (not a guarantee by any means) was the middle of December.

Obviously it isn't directly feasible to dedicated all our resources to one specific feature at a time and we need to be able to scale our workflow to the developers we have, and topics like the API keys and Draft/Publish are simple in their development requirements so we wanted to get these finished up and released as we continue the planning process for the new plugin API as well as the database layer (https://portal.productboard.com/strapi/1-public-roadmap/c/73-database-layer-v4)
